templates/homebrew.mxcl.buildkite-agent.plist.j2

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
```

Comment:
```suggestion

<!-- https://launchd.info -->

<plist version="1.0">
```

Reply:
Since macOS launchd is full of inscrutable pitfalls in my so-far-quite-painful experience :(